onnxruntime/hosting/environment.cc

```diff
@@ -10,33 +10,31 @@
 namespace onnxruntime {
 namespace hosting {
 
-HostingEnvironment::HostingEnvironment(onnxruntime::logging::Severity severity) : logger_id_("HostingApp"),
-                                                                                  default_logging_manager_(
-                                                                                      std::unique_ptr<onnxruntime::logging::ISink>{&sink_},
-                                                                                      severity,
-                                                                                      /* default_filter_user_data */ false,
-                                                                                      onnxruntime::logging::LoggingManager::InstanceType::Default,
-                                                                                      &logger_id_) {
-  auto status = onnxruntime::Environment::Create(this->runtime_environment_);
+HostingEnvironment::HostingEnvironment(logging::Severity severity) : severity_(severity),
+                                                                     default_filter_user_data_(false),
```

Comment:
If there is no chance to set it to true, you could just use `false` in `GetLogger()` and avoid this private property.